sirepo/uri_router.py

```diff
@@ -213,6 +213,28 @@ def sr_set_log_user(self, log_user):
 
     class _WebSocket(websocket.WebSocketHandler):
         async def on_message(self, msg):
+            import asyncio
```

Comment:
Why? We import asyncio in sirepo.util which is imported everywhere so I can't imagine keeping the import local here is helpful.

Reply:
82da885